cli/cli/cli.go

```diff
@@ -148,17 +145,17 @@ func (curveadm *CurveAdm) init() error {
 	dbpath := fmt.Sprintf("%s/curveadm.db", curveadm.dataDir)
 	s, err := storage.NewStorage(dbpath)
 	if err != nil {
-		log.Error("Init SQLite database failed",
-			log.Field("Error", err))
+		zaplog.LOG.Error("Init SQLite database failed",
```

Comment:
You can avoid directly exposing the variable LOG by adding functions in zaplog.go, which can also be used in the same way as the original, such as zaplog.Info(), zaplog.Error() and other functions.

Reply:
done